### ISSUE TITLE
What are you supposed to do with this call?

### ISSUE THREAD

**Issue by morningtundra** [vkarpov15/thecodebarbarian.com#46]
`window.oauthOpen = require('oauth-open');`

What are you supposed to do with this call?

`require` doesn't work in the client and this doesn't belong in the auth-server.js.

Article implies it goes in your index.html but this gives

`Uncaught ReferenceError: require is not defined`

Can you clarify?

**Comment by vkarpov15** [comment]
I presume you're asking about this blog post: http://thecodebarbarian.com/oauth-with-node-js-and-express.html

Here's a quote from the above blog post that should answer your question:

The https://codebarbarian-images.s3.amazonaws.com/open.dist.js file in the above example is a Webpack bundle of the below script. I pre-compiled that script for convenience so the code in this article doesn't require a compiler.

```javascript
window.oauthOpen = require('oauth-open');
```

In other words, the `window.oauthOpen ...` code you pasted is what Webpack compiles into the `https://codebarbarian-images.s3.amazonaws.com/open.dist.js` file that the first example in this section uses: http://thecodebarbarian.com/oauth-with-node-js-and-express.html#client-app-implementation
